app/models/site_config.rb

```diff
@@ -26,7 +26,8 @@ class SiteConfig < RailsSettings::Base
 
   # Authentication
   field :allow_email_password_registration, type: :boolean, default: false
-  field :allow_email_password_login, type: :boolean, default: true
+  field :allow_email_password_login, type: :boolean, default: false
```

Comment:
I didn't catch this on the first review, but I just noticed this now -- I'm curious [why this is changing](https://github.com/forem/forem/pull/10838/files#diff-68da686a0f7348bf9a2c59e669043f21019ab8dfb6363cea6daa534c5c8296d7R26) to `false`? @forem/creator-onboarding intentionally will need this to be `true` for in order to [ensure that email login defaults to being "on"](https://github.com/forem/InternalProjectPlanning/issues/91) for newly-created forems.

Reply:
No worries, I will change it back to `true`. I had it that way to test the feature I was building 😄

Reply:
Thanks, @msarit! Just for more context for you + @joshpuetz + @lisasy...

@forem/creator-onboarding is making changes to SiteConfig this cycle in order to make the forem creator onboarding process as smooth as possible. So, if you end up needing to make any changes to that file, please do drop us a line 🙏 We're trying our best to keep tabs on all the SiteConfig-related PRs, but there have been a lot this cycle! So if you end up changing any "defaults" in SiteConfig there is a good chance that it will affect our cycle work, and pinging us directly would be super helpful :)